opencga-app/app/scripts/azure/arm/azuredeploy.json

```diff
@@ -85,6 +91,24 @@
             "metadata": {
                 "description": "The password for the 'admin' user on the Avere cluster configuration portal."
             }
+        },
+        "webserversSSHKeyData": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The RSA data for the SSH Public Key used to authenticate to Avere nodes."
+            }
+        },
+        "tomcatDockerImage" : {
+            "type": "string",
+            "metadata": {
+                "description": "The docker image and tag for running the tomcat service."

```

Comment:
Appreciate the description is pretty clear but would be good to have an example string that meets the expected format on these i.e. repo/image:tag

Reply:
Deal, will add it